Resources/Prototypes/_Sunrise/Catalog/uplink_catalog.yml

```
@@ -624,6 +664,25 @@
       tags:
       - NukeOpsUplink
 
+- type: listing
+  id: UplinkWeaponDragunov
+  name: uplink-weapon-dragunov-name
+  productEntity: ClothingBackpackDuffelSyndicateFilledDragunov
+  description: uplink-weapon-dragunov-desc
+  icon: { sprite: _Sunrise/Objects/Weapons/Guns/Snipers/dragunov/big.rsi, state: icon }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)